### ISSUE TITLE
Is it possible add comment to entry in HAR if request/response filter triggered?

### ISSUE THREAD

**Issue by Balinski** [lightbody/browsermob-proxy#657]
Is it possible add comment to entry in HAR if request/response filter triggered?
EXAMPLE:
Charlees add next commnet to entry if request/response filter triggered: 
"comment": "Rewrite Tool: status changed to \"404 Server Error\""

Thanks!

**Comment by Balinski** [comment]
Hello!
Any updates?

**Comment by jekh** [comment]
No, it's not currently possible. I really want this to be possible, but I haven't figured out a way to cleanly expose the HAR to filters yet. It is one of the key requirements of the new REST API that is still brewing, so I'll add tag this issue and update it when there is progress.

**Comment by Balinski** [comment]
Thanks for your reply!
